system-variables.md

```diff
@@ -759,28 +812,40 @@ Constraint checking is always performed in place for pessimistic transactions (d
 > - Since v5.4.0, for a newly deployed TiDB cluster, this variable is enabled by default.
 
 - Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
 - Default value: `ON`
 - This variable is used to control whether to enable the index merge feature.
 
+### tidb_enable_legacy_instance_scope <span class="version-mark">New in v6.0.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Default value: `ON`
+- This variable permits `INSTANCE` scoped variables to be set with `SET SESSION` as well as `SET GLOBAL` syntax.
+- This option is enabled by default for compatibility with earlier versions of TiDB.
```

Comment:
This was added to 6.0 but not documented. It is the new feature being documented in this PR. @CbcWestwolf will translate this to Chinese.